base/utf16.jl

```diff
@@ -110,7 +110,9 @@ function is_valid_utf16(data::AbstractArray{UInt16})
     return i > n || !utf16_is_surrogate(data[i])
 end
 
-is_valid_utf16(s::UTF16String) = is_valid_utf16(s.data)
+isvalid(str::UTF16String) = isvalid(UTF16String, str.data)
+is_valid_utf16(str::UTF16String) = isvalid(UTF16String, str.data)
+is_valid_utf16(data::AbstractArray{UInt16}) = isvalid(UTF16String, data)
```

Comment:
OK, so all trace of the old method goes away (except in deprecated.jl), right?
I'd already added them to deprecated.jl, just didn't know I could get rid of the definitions in utf16.jl...
Thanks!